internal/http/server.go

```diff
@@ -169,6 +175,18 @@ func (srv *Server) UseShutdownTimeout(d time.Duration) *Server {
 	return srv
 }
 
+// UseIdleTimeout configure idle connection timeout
+func (srv *Server) UseIdleTimeout(d time.Duration) *Server {
```

Comment:
I'd expect these to be called from somewhere?

Reply:
Oh yes I will add those..